shared/utils/domains.js

```diff
@@ -60,8 +60,14 @@ export function stripSubdomain(hostname: string) {
 
 export function isCustomSubdomain(hostname: string) {
   const parsed = parseDomain(hostname);
-  if (!parsed) return false;
-  if (!parsed.subdomain || parsed.subdomain === "www") return false;
+  if (
+    !parsed ||
+    !parsed.subdomain ||
+    parsed.subdomain === "app" ||
```

Comment:
🍖  Do not consider `app.` to be a team subdomain, it will be used for the root login and `www.` will be used for marketing site.